src/components/manifold-subscription-list/manifold-subscription-list.tsx

```diff
@@ -64,13 +70,40 @@ export class ManifoldSubscriptionList {
     this.getSubscriptions(this.owner);
   }
 
+  handleCtaClick = (subId: string, href: string) => (e: MouseEvent) => {
+    e.preventDefault();
+    this.ctaClick.emit({ id: `manifold-cta-sub-${subId}` });
```

Comment:
I see that you're copying the format from plan-table, and TBH I have no idea why they're using that format! The purpose of these events is so that the user can do something like make a SPA navigation, and they'd use the event data to construct those URLs. So in this case I think we should do something more like this:

```ts
this.ctaClick.emit({ subscriptionId: subId });
```